### DISCUSSION TITLE
3 of the same github workflows is running at the same time

### DISCUSSION THREAD

**Discussion by nkstonks** [community/community#27122] [Actions]
<p>I have this workflow that publishes my python package to PyPI, but for some reason, GitHub runs the workflow 3 times (in parallel) so only one of them (the one that started the earliest) manages to succeed and the other ones fail.<br>
It’s really triggering me and I want to fix it real bad, but I don’t know what I did wrong.</p>
<p>The link to the repo is here: <a href="https://github.com/nkstonks/string-utils" class="inline-onebox" rel="noopener nofollow ugc">GitHub - nkstonks/string-utils</a></p>
<p>The link to the workflow is here: <a href="https://github.com/nkstonks/string-utils/blob/c07b62e875ff81e633240aff4169ed37aff3bf37/.github/workflows/python-publish.yml" rel="noopener nofollow ugc">https://github.com/nkstonks/string-utils/blob/c07b62e875ff81e633240aff4169ed37aff3bf37/.github/workflows/python-publish.yml</a></p>
<p>Please help me, thanks.</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>Hey the workflow file you linked to doesn’t exist any more. Are you still having this problem? If so, could you provide some updated links so we can see the file?</p>

**Comment by nkstonks** [comment]
<p>yeah, about that, I eventually was able to fix it. Thanks for checking in, I’ll mark this as solved.</p>

**Comment by mjpriyanshu** [comment]
I started with Python because it was beginner-friendly and now I'm exploring Java.
